lms/static/sass/views/_shoppingcart.scss

```diff
@@ -56,6 +56,8 @@ $light-border: 1px solid $gray-l5;
 }
 
 .confirm-enrollment {
+  @include direction();
```

Comment:
I don't see this in the documentation, so I'm not sure what role it serves here. I'm cc'ing @frrrances in case she's more familiar with this.